docs/dev/shop-setup/GitLab_Gold.md

```diff
@@ -0,0 +1,102 @@
+# GitLab Gold Shop Setup:
+
+## Option 1: Subgroup of LibreFoodPantry (inherits Gold features from LFP group)
+
+### Setting up groups and user permissions:
+
+![GitLab Gold Option 1 Users & Permissions Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Gold_Setup_Option1_Users_And_Permissions.png)
+
+#### Trustee:
+
+1. Add the shop manager to the LFP group with maintainer permissions
+
+2. Create a subgroup in LFP for the shop
+
+3. Add the shop manager to the shop subgroup as an owner
+
+4. Add the shop developers to the parent LFP group with reporter permissions
+
+#### Shop Manager:
+
+1. Add the shop developers to the shop subgroup with developer permissions
+
+### Setting up the shop issue board:
+
+![GitLab Shop Board Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Setup_Shop_Board.png)
+
+#### Shop Manager:
+
+1. Create a new shop-level issue board
+
+2. Add a new column by clicking "Add list" and create a new group label for (and column) for each of the following (in order on the board from left to right):
+    1. Backlog
+    2. In Sprint
+    3. Needs Review
+    4. Needs Merge
+    5. Done
+
+- You may need to click on the label in the add list menu for the list to be added to the board    
+
+### Setting up the project:
+
+![GitLab Gold Option 1 Groups & Projects Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Gold_Setup_Option1_Groups_And_Projects.png)
+
+#### Shop Manager:
+
+1. Fork the repository you will be working on from the LFP group to the shop 
+
+
+
+## Option 2: Standalone Group
+
+### Setting up groups and user permissions:
+
+![GitLab Gold Standalone Group Users & Permissions Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Standalone_Group_Setup_Users_And_Permissions.png)
+
+#### Trustee:
+
+1. Add the shop manager to the LFP group with maintainer permissions
+
+2. Add the shop developers to the LFP group with reporter permissions
+
+#### Shop Manager:
+
+1. Create a GitLab group for the shop
+
+2. Add the shop developers to the shop group with developer permissions
+
+### Setting up the shop issue board:
+
+![GitLab Shop Board Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Setup_Shop_Board.png)
+
+#### Shop Manager:
+
+1. Create a new shop-level issue board
```

Comment:
Same questions about where the board goes and what permissions are needed by shop-members on what?

If the board is in the shop, then do shop members need reporter on LFP?

Now that I'm writing this, I'm seeing the need for reporter status on LFP... at least on the projects they are working on. This will let them groom issues (e.g., labels, assignment, etc.). But then why not have the shop board in LFP too? In fact, I think merge requests officially exist in LFP and not in the shop fork. That means the shop board needs to be in LFP. I think team boards would also be better placed in LFP because they would not need a separate issue to represent the story they are working on, they would just add the card to their board too.

Of course all of this means that Trustees have to grant permissions for all shop members; ugh. What's the least permission we can grant to Shop managers that would allow them to grant permissions to shop members?

Reply:
Only users with owner permissions can grant permissions or add members to a group: https://docs.gitlab.com/ee/user/permissions.html#group-members-permissions

Reply:
I'm leaving this as-is for now, when we make a decision on the boards I'll update it.

Reply:
That's fine. I'll consider this resolved.